src/components/LinksList/index.js

```diff
@@ -18,22 +21,21 @@ class LinksList extends React.Component {
     } = this.props;
     return (
       <div>
-      { selectedMoc && selectedMoc.govtrack_id ? 
-        <Card 
-          className="actions-section" 
-          title="Actions:" 
-          extra={<div> <Button href="#" onClick={() => handleAutoFillMember(selectedMoc.govtrack_id)}>AutoFill Form</Button> <Button  onClick={() => this.props.showConfirm()}>No Events</Button></div>} 
-          style={{ width: 'auto' }}
-        /> : 
-        <div></div>
-      }
+        { selectedMoc && selectedMoc.govtrack_id && 
+        (<Card
+            className="actions-section"
+            title="Actions:"
+            extra={<div> <Button href="#" onClick={() => handleAutoFillMember(selectedMoc.govtrack_id)}>AutoFill Form</Button> <Button onClick={() => this.props.showConfirm()}>No Events</Button></div>}
+            style={{ width: 'auto' }}
+          />)
+        }
         <List
           size="small"
           locale={{
-            emptyText: 'Member links will appear here.'
+            emptyText: 'Member links will appear here.',
           }}
-          header={<div><strong>{selectedMoc && selectedMoc.member_name ? <div>{selectedMoc.member_name} Links</div> : ''}</strong></div>}
-          footer={<div>{selectedMoc && selectedMoc.member_name ? <Tooltip title="Add Link"><Button onClick={() => showModal('add')} type="primary" size="small" style={{ marginLeft: '50%', transform: 'translateX(-50%)' }}><Icon type="plus-circle" /></Button></Tooltip> : ''}</div>}
+          header={<div><strong>{selectedMoc && selectedMoc.displayName ? <div>{selectedMoc.displayName} Links</div> : ''}</strong></div>}
```

Comment:
trying to be more consistent with attributes. Also front end javascript variables should be camel cased.